src/support/utilities.h

```diff
@@ -30,6 +30,8 @@
 
 namespace wasm {
 
+using vec128_t = uint8_t __attribute((vector_size(16)));
```

Comment:
This seemed cute but probably isn't super portable. Happy to use a `std::array<uint8_t, 16>` or similar instead.

Reply:
Not my area of expertise, but I see `__m128` in portable vector-using codebases.

Reply:
I don't expect that this will work on Windows. Not sure about `__m128` but IIRC that's just int128, right? That doesn't seem much more useful than an array of 16 chars?